### DISCUSSION TITLE
Does GitHub Sponsors Accept Apple Pay?

### DISCUSSION THREAD

**Discussion by aaronhma** [community/community#120585] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Does GitHub Sponsors Accept Apple Pay?

**Comment by Visual-Studio-Coder** [comment]
I wish I could, but I'm pretty sure they don't let you. Hopefully Apple jumps in on this idea one day

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
